src/app/content/routes.ts

```diff
@@ -5,7 +5,15 @@ import { SelectedResult } from './search/types';
 import { Params } from './types';
 
 const CONTENT_PATH = '/books/:book/pages/:page';
-const VERSIONED_CONTENT_PATH = '/books/:book([^@/]+)@:version/pages/:page';
+const UUID_CONTENT_PATH =
```

Comment:
this might read better with the uuid separated out like:

```
const MATCH_UUID = '[\\da-z]{8}-[\\da-z]{4}-[\\da-z]{4}-[\\da-z]{4}-[\\da-z]{12}';
const UUID_CONTENT_PATH =  `/books/:uuid(${MATCH_UUID})@:version/pages/:page`;
```